Greenify-FrontEnd/app/components/Login.js

```diff
@@ -65,7 +65,7 @@ export default class Login extends React.Component {
   }
   login = () => {
     if(this.state.username.length && this.state.password.length !==0){
-      fetch("http://192.168.1.109:3000/login",{
+      fetch("http://192.168.1.83:3000/login",{
```

Comment:
Things like this (an IP address) are going to vary depending on whose development environment the app is running in. To keep this value from changing in source control every time a commit is made in a different environment, it helps to keep values like this in a configuration file which is not tracked by source control. Here's a package I've used in the past to handle configuration options like this: [https://www.npmjs.com/package/dotenv](url)